modules/YetiDriver/powermeter/powermeterImpl.cpp

```diff
@@ -54,12 +54,15 @@ void powermeterImpl::init() {
 void powermeterImpl::ready() {
 }
 
-std::string powermeterImpl::handle_stop_transaction(std::string& transaction_id) {
-    return "NOT_SUPPORTED";
+types::powermeter::TransactionStopResponse powermeterImpl::handle_stop_transaction(std::string& transaction_id) {
+    return {.status = types::powermeter::TransactionRequestStatus::NOT_SUPPORTED,
```

Comment:
The designated initializers for structs (the dot notation) is valid C99 afaik but not in C++17 (it was added in C++20 I think). GCC supports a lot of C99 while compiling C++ but this may not be portable when using other compilers